lib/generate.js

```diff
@@ -43,7 +61,10 @@ async function generate(text, layout, info) {
     .run(rawTextMdast);
 
   // Add classes to the special sections in the mdast
-  const taggedTextMdast = tagCalloutSections(textMdastWithIds);
+  tagCalloutSections(textMdastWithIds);
+
+  // Reindex the footnote link text to match the footer
```

Comment:
Why don’t we just use inline footnotes? https://unifiedjs.com/explore/package/remark-footnotes/#use

Then we don’t have to worry about re-indexing them.

Reply:
Because then they can’t be multiline and they make the markdown harder to follow. It’d probably be less bad if we used the footnote-style link format, I’ll give that a try

Reply:
TBQH, I just don't like inline footnotes; I think they make the text look messy. We might be able to handle them if we did this in the hast instead of the mdast.

Reply:
OK, based on the latest commit, I got it working for both inline and postfix footnotes by walking the hast.